src/BitStringAddresses/bitstring.jl

```diff
@@ -0,0 +1,303 @@
+"""
+    num_chunks(::Val{B})
+
+Determine the number of chunks needed to store `B` bits.
+"""
+function num_chunks(::Val{B}) where {B}
+    return if B ≤ 0
+        throw(ArgumentError("`B` must be positive!"))
+    else # Turns out it's never worth using anything other than UInt64 here.
+        return (B - 1) ÷ 64 + 1
```

Comment:
Is this function used a lot, i.e. in hot loops? If so it might be worth checking whether replacing `÷ 64` by `>>> 0x6` might save a few femtoseconds.